tekton/src/tasks/gitops/gitops-mcsp-cluster.yml.j2

```diff
@@ -0,0 +1,124 @@
+    - name: github_pat
+      type: string
+    - name: git_commit_msg
+      type: string
+    - name: artifactory_username
+      type: string
+      default: ""
+    - name: artifactory_token
+      type: string
+      default: ""
+    - name: avp_aws_access_key
+      type: string
+    - name: avp_aws_secret_key
+      type: string
+    - name: avp_aws_secret_region
+      type: string
+    - name: argocd_url
+      type: string
+    - name: argocd_username
+      type: string
+    - name: argocd_password
+      type: string
+
+    - name: mas_catalog_version
+      type: string
+    - name: mas_catalog_image
+      type: string
+
+    - name: common_services_channel
+      type: string
+  stepTemplate:
+    name: gitops-mcsp-cluster
+    env:
+      - name: CLUSTER_NAME
+        value: $(params.cluster_name)
+      - name: ACCOUNT_ID
+        value: $(params.account)
+      - name: GIT_BRANCH
+        value: $(params.git_branch)
+      - name: GITHUB_ORG
+        value: $(params.github_org)
+      - name: GITHUB_HOST
+        value: $(params.github_host)
+      - name: GITHUB_REPO
+        value: $(params.github_repo)
+      - name: GIT_COMMIT_MSG
+        value: $(params.git_commit_msg)
+      - name: ARTIFACTORY_USERNAME
+        value: $(params.artifactory_username)
+      - name: ARTIFACTORY_TOKEN
+        value: $(params.artifactory_token)
+      - name: SM_AWS_ACCESS_KEY_ID
+        value: $(params.avp_aws_access_key)
+      - name: SM_AWS_SECRET_ACCESS_KEY
+        value: $(params.avp_aws_secret_key)
+      - name: SM_AWS_REGION
+        value: $(params.avp_aws_secret_region)
+      - name: GITHUB_USER
+        value: $(params.github_user)
+      - name: GITHUB_PAT
+        value: $(params.github_pat)
+      - name: ARGOCD_URL
+        value: $(params.argocd_url)
+      - name: ARGOCD_USERNAME
+        value: $(params.argocd_username)
+      - name: ARGOCD_PASSWORD
+        value: $(params.argocd_password)
+
+      - name: MAS_CATALOG_VERSION
+        value: $(params.mas_catalog_version)
+      - name: MAS_CATALOG_IMAGE
+        value: $(params.mas_catalog_image)
+
+      - name: COMMON_SERVICES_CHANNEL
+        value: $(params.common_services_channel)
+  steps:
+    - args:
+      - |-
+          git config --global user.name "MAS Automation"
+          git config --global user.email "${GITHUB_USER}"
+          git config --global user.password "${GITHUB_PAT}"
+
+          mkdir -p /tmp/init-cluster
+
+          mas gitops-mcsp-cluster \
+            -a "${ACCOUNT_ID}" \
+            -c "${CLUSTER_NAME}" \
+            -r "${SM_AWS_REGION}" \
+            --dir /tmp/init-cluster \
+            --github-push \
+            --github-host $GITHUB_HOST \
+            --github-org  $GITHUB_ORG \
+            --github-repo $GITHUB_REPO \
+            --git-branch $GIT_BRANCH
+      command:
+        - /bin/sh
+        - -c
+      name: gitops-mcsp-cluster
+      imagePullPolicy: Always
+      image: quay.io/ibmmas/cli:7.7.0-pre.mascore962
```

Comment:
can you change these images to be gitops?

Reply:
done